File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        
     LinkedList<Integer> stack=new LinkedList<>();
     stack.push(nums[0]);
        
     if(nums.length==1)
         return 1;
        
        for(int i=1;i<nums.length;i++)
        {
            
            if(stack.peek()!=nums[i])
            {
                stack.push(nums[i]);
                nums[stack.size()-1]=stack.peek();
            }
            
        }
        
        return stack.size();
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.kt
class Solution {
    fun removeDuplicates(nums: IntArray): Int {
        var res=0
        val length=nums.size
        for(i in 1 until length){
            if(nums[i]!=nums[res]){
                nums[res+1]=nums[i]
                res++
            }
        }
        return res+1
    }
}